src/interfaces/wallet.h

```diff
@@ -204,7 +204,7 @@ class Wallet
         int& num_blocks) = 0;
 
     //! Fill PSBT.
-    virtual std::optional<common::PSBTError> fillPSBT(int sighash_type,
+    virtual std::optional<common::PSBTError> fillPSBT(std::optional<int> sighash_type,
```

Comment:
Since there is now a `PSBTError::OK` it seems like there is no need for using an `std::optional<common::PSBTError>` return type. But this can be made more consistent in a follow-up.

Reply:
Will leave for a followup